.gitignore

```diff
@@ -0,0 +1 @@
+.idea/
```

Comment:
What's this about?

Reply:
A folder that Pycharm creates.

Reply:
As not everyone in the team uses PyCharm, can I suggest that you create a global .gitignore for your own system, and leave the repo one as-is? See https://help.github.com/articles/ignoring-files/#create-a-global-gitignore :)

Reply:
Great idea! Thanks.